.github/workflows/main.yml

```diff
@@ -56,16 +66,16 @@ jobs:
         with:
           browser: chrome
           build: npm run build:emulator
-          start: npm run start:emulator
-          wait-on: 'http://localhost:3021'
+          start: npm run dev:emulator
```

Comment:
Why are we running cypress with dev instead of build->start?

Reply:
Cause the instance run in production mode, it wont run tests on the emulators. (Have already tried to change NODE_ENV but couldnt really find a way around it)

Reply:
Ahh I see, yeah all g